### PR TITLE
feature/6108-Dylan-CombineJSONToCommon

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
Moved all strings in health.json, payments.json, home.json to common.json and sorted alphabetically
Removed health.json, payments.json, and home.json 'title' strings and update accordingly to 'health' 'payments.title' and 'home' that already existed in common.json

removed any usage of tc, th, or tp for translations and they are now all t
removed any string usage of 'common: or 'home: or 'payments: or 'health: - this is because it is now all in one file
any functions that used translation updated accordingly

## Screenshots/Video
N/A

## Testing
yarn test and sniff tests

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Move lingering health/home/payments .json files into common.json.

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx [hunk 1]
```diff
@@ -5,27 +5,27 @@ import { formatDateUtc } from 'utils/formattingUtils'
 
 /**
  * Returns the display text and accessibilityLabel value for rxNumber used in Prescriptions. Defaults to 'None noted' if not given.
- * @param healthT - translation set with `useTranslation(NAMESPACE.HEALTH)`
+ * @param healthT - translation set with `useTranslation(NAMESPACE.COMMON)`
```
Comment (theodur) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx:8]:
```suggestion
 * @param t - translation set with `useTranslation(NAMESPACE.COMMON)`
```

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx [hunk 2]
```diff
@@ -5,27 +5,27 @@ import { formatDateUtc } from 'utils/formattingUtils'
...
+  const noneNoted = t('noneNoted')
+  const rxNumber = `${t('prescription.rxNumber')} ${prescriptionNumber || noneNoted}`
+  const rxNumberA11yLabel = `${t('prescription.rxNumber.a11yLabel')} ${a11yLabelID(prescriptionNumber) || noneNoted}`
   return [rxNumber, rxNumberA11yLabel]
 }
 
 /**
  * Returns the display text and accessibilityLabel value for date used in Prescriptions. Defaults to 'None noted' if not given.
- * @param healthT - translation set with `useTranslation(NAMESPACE.HEALTH)`
+ * @param healthT - translation set with `useTranslation(NAMESPACE.COMMON)`
```
Comment (theodur) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx:22]:
```suggestion
 * @param t - translation set with `useTranslation(NAMESPACE.COMMON)`
```

**Review by theodur — COMMENTED** [review]
I can only imagine how worn out the Cmd + F buttons are on your keyboard  😅. Changes look good, made a few minor suggestions on some param names in a couple DocBlocks. It looks like there are some merge conflicts though

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx (hunk 2 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx]: committed

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx (hunk 1 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionCommon/PrescriptionUtils.tsx:8]: commited

**Comment by Sparowhawk** [comment]
> I can only imagine how worn out the Cmd + F buttons are on your keyboard 😅. Changes look good, made a few minor suggestions on some param names in a couple DocBlocks. It looks like there are some merge conflicts though

fixed, yeah, it was rough finding all of those in each file lol

**Review by theodur — APPROVED** [review]
I resolved the merge conflicts for this, should be good now

**Review by theodur — DISMISSED** [review]

File: VAMobile/src/components/NavigationTabBar.tsx [hunk 3]
```diff
@@ -74,8 +74,7 @@ const NavigationTabBar: FC<NavigationTabBarProps> = ({ state, navigation, transl
           const isFocused = state.index === index
           const routeName = route.name.replace('Tab', '')
           const lowerCaseRoute = routeName.toLowerCase()
-          // TODO: remove this conditional once all tab names have been moved to common.json
-          const translatedName = ['payments', 'benefits'].includes(lowerCaseRoute) ? translation(`common:${lowerCaseRoute}.title`) : translation(`${routeName.toLowerCase()}:title`)
+          const translatedName = translation(`${lowerCaseRoute}.title`)
```
Comment (dumathane) [inline VAMobile/src/components/NavigationTabBar.tsx:77]: niceeeee.

Comment (Sparowhawk) [reply VAMobile/src/components/NavigationTabBar.tsx:77]: yeah that was ugly looking before lol

**Review by dumathane — APPROVED** [review]

**Review by dumathane — APPROVED** [review]

**Review by TimRoe — APPROVED** [review]
Approving in [validation of the latest commit making copy updates to move along](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/6505/commits/5c80c3269058e4cad60c75a390fbb3b421e3a6bc); did not look at previously approved changes that Theo/Binny established as satisfactory for move to QA.

**Pull request merged**